src/azure-cli/azure/cli/command_modules/cdn/tests/latest/test_custom_domain_scenarios.py

```diff
@@ -162,10 +161,8 @@ def test_cdn_custom_domain_https_verizon(self, resource_group):
 
         checks = [JMESPathCheck('name', custom_domain_name),
                   JMESPathCheck('hostName', hostname),
-                  JMESPathCheck('customHttpsParameters.protocolType', 'IPBased'),
-                  JMESPathCheck('customHttpsParameters.certificateSource', 'Cdn'),
-                  JMESPathCheck('customHttpsParameters.minimumTlsVersion', 'None'),
-                  JMESPathCheck('customHttpsParameters.certificateSourceParameters.certificateType', 'Shared')]
+                  JMESPathCheck('customHttpsProvisioningState', 'Enabling'),
```

Comment:
Could you share the context about removing these checks?

Reply:
A recent service change makes these fields no longer available on the return value.